hitl-setup/setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: 2024 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FORCE="$1"
+set -exuo pipefail
```

Comment:
This is a script run by a user. It is my strong belief that `set -e` only really has a place in either simple commands (and no invocations of shell functions) or in CI scripts. Everything else is asking for trouble.

I can rewrite it to not use `set -e` unwisely.

Reply:
Although frankly this example is also a reason why we maybe shouldn't even do it in CI. In CI, we think, "well if it exits with non-zero exit code when it should have run fine that's no big problem, we'll just fix the script there and then". But it can also happen the other way around: you think it will exit but it continues running.

```bash
set -e
f() { false; echo "This won't run, right?"; }
if f; then  # same with: f && echo survived
    echo survived
fi
```
prints
```
This won't run, right?
survived
```

Reply:
If you think is should be rewritten, please do so.
I did the `set -e` mostly because I didn't want to put ` || exit 1` behind all the calls to systemctl, and I think it should work ok for that as is.

There are no functions, and the `for INTERFACE` and `if` block end with explicit `exit` statement.
Which I assumed would be ok, but I definitely don't know all the intricacies involved.